lib/gringotts/gateways/authorize_net.ex

```diff
@@ -352,9 +347,10 @@ defmodule Gringotts.Gateways.AuthorizeNet do
   """
   @spec store(CreditCard.t, Keyword.t) :: tuple
   def store(card, opts) do
-    request_data = cond  do
-      opts[:customer_profile_id] -> card |> create_customer_payment_profile(opts) |> generate
-      true -> card |> create_customer_profile(opts) |> generate
+    request_data = if opts[:customer_profile_id] do
```

Comment:
can we use case, coz ultimately its going to be converted to case ?

Reply:
Why `case` isn't this a simple `if`? And I think this change was suggested by credo (48d7bd84)

Reply:
Actually I just need to check whether customer_profile_id is present that's why not `case` because yes `credo` warns that there should be **more than two conditions** to match on if you use `case`.